include/dxc/DxilContainer/DxilRuntimeReflection.h

```diff
@@ -12,7 +12,7 @@
 #pragma once
 
 #include "dxc/DXIL/DxilConstants.h"
-#include "dxc/WinAdapter.h"
+#include "dxc/Support/WinIncludes.h"
```

Comment:
Looks like this was a way to allow the include to remain after DxiLConstants?

Reply:
The ordering here doesn't matter. DxilRuntimeReflection.h depends on the windows headers which are from WinIncludes not WinAdapter. We got lucky by include ordering previously.